### DISCUSSION TITLE
using git to track changes to any/all schema's defined in Hasura

### DISCUSSION THREAD

**Discussion by geckotechcr** [hasura/graphql-engine#7358] [Question] [answered]
Hello, I am new to Hasura and have set up a docker of the core 2.x version.
It is a great tool for doing graphql api proof of concept build and testing.  The UI also allows for live schema changes.
My question is this:
Are there plans to integrate GIT into hasura CORE to allow self-hosted hasura instance to commit and track a specific schema and its changes?

**Comment by rikinsk** [comment, marked as answer]
Seems like Hasura migrations should help you with this. [Docs here](https://hasura.io/docs/latest/graphql/core/migrations/index.html)
